### PR TITLE
{AKS} Support reset default value for loadbalancer profile and natgateway profile

### PR DESCRIPTION
**Pull request by MartinForReal**
**Related command**

`az aks update --load-balancer-outbound-ports 0`

**Description**
related: #27071
allow customer to set --load-balancer-outbound-ports, --nat-gateway-managed-outbound-ip-count, --load-balancer-managed-outbound-ipv6-count to 0

**Testing Guide**

create a aks cluster and run following command

`az aks update --load-balancer-outbound-ports 0`
**History Notes**

[AKS] `az aks update`: Zero can be set to outbound-ports,outbound-ip-count in loadbalancer profile and outbound-ip-count in natgateway profile in aks

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by FumingZhang — COMMENTED** [review]
lgtm

Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=86661153&view=results) to validate the change

**Review by yanzhudd — APPROVED** [review]

**Comment by yanzhudd** [comment]
CLI is on a release process and codes cannot be merged into the main repo. After we finish the release, will merge it.

**Comment by yanzhudd** [comment]
Hi @MartinForReal 
it seems that this PR causes our CI to block, could you please help to take a look?
https://dev.azure.com/azclitools/public/_build/results?buildId=133826&view=logs&j=c4905e22-e505-5bfe-aa0a-49d3f9325fa2&t=cf247c3b-aad8-52b9-12ca-ceb3e5942fcf&l=2998

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_create_loadbalancer_then_update.yaml
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_create_loadbalancer_then_update.yaml:628]: This PR adds 2 `GET` requests and 1 `PUT` request for `https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.ContainerService/managedClusters/cliakstest000002`

#28407 (merged on 2024-02-21) bumped the API version of `/providers/Microsoft.ContainerService/managedClusters` from `2023-11-01` to `2024-01-01` in the existing requests.

This PR's CI was run on 2024-01-30 and passed, but it is not re-run after #28407. As adding new lines won't cause merge conflict, this PR can be merged, but triggers CI failure after merging.

**Pull request merged**
